traitsui/qt4/video_editor.py

```diff
@@ -233,10 +251,19 @@ def init(self, parent):
         )
         self.control.setBackgroundRole(QPalette.ColorRole.Window)
 
-        self.media_player = QMediaPlayer(None, QMediaPlayer.VideoSurface)
+        if is_qt5:
+            self.media_player = QMediaPlayer(None, QMediaPlayer.VideoSurface)
+        else:
+            self.media_player = QMediaPlayer()
         self._set_video_url()
         self.media_player.setVideoOutput(self.control)
-        self.media_player.setMuted(self.muted)
+        if not is_qt5:
+            from pyface.qt.QtMultimedia import QAudioOutput
+            self._audio = QAudioOutput()
+            self._audio.setMuted(self.muted)
```

Comment:
`Editor` doesn't derive from `HasStrictTraits` I guess?

Reply:
No, almost nothing does in TraitsUI (you're lucky if it doesn't derive from `HasPrivateTraits`). But yes, I should add a trait for this (probably just an `Any` for now).